controllers/articles.controllers.js

```diff
@@ -24,4 +24,21 @@ exports.getArticles = (req, res, next) => {
     .catch((err) => {
         next(err);
     });
+}
+
+exports.getCommentsByArticleId = (req, res, next) => {
+    const { article_id } = req.params;
+
+    selectArticleById(article_id)
+    .then(() => {
+        selectCommentsByArticleId(article_id)
+        .then((comments) => {
+            res.status(200).send({comments});
+        })
+    })
+    .catch((err) => {
+        next(err);
+    });
+
```

Comment:
This is a good start, but we are currently having to always wait for the selectArticleById to happen first and then querying for all of its comments, can you think of a way to do both of these two things at the same time?